README.md

```diff
@@ -1,18 +1,19 @@
-# Alert Logic Agent Container for Kubernetes, ECS, and Docker
+# Alert Logic Agent Container for Kubernetes, ECS, EB, and Docker
```

Comment:
Small detail, but shall we spell out "Elastic Beanstalk" here since it's the title?

Reply:
agree, EB is not popular enough!

Reply:
According to AWS, it's "AWS Elastic Beanstalk" on the first reference, and "Elastic Beanstalk" going forward from there. I can perform an edit for grammar and for AL Info Dev style standards.